src/Apps/Partner/PartnerApp.tsx

```diff
@@ -54,7 +54,10 @@ export const PartnerApp: React.FC<PartnerAppProps> = ({
 
         <FullBleed mb={[2, 4]}>
           {firstEligibleBadgeName ? (
-            <Marquee speed="static" marqueeText={firstEligibleBadgeName} />
+            <Marquee
+              speed="static"
+              marqueeText={firstEligibleBadgeName.replace(" ", "-")} // hypenate gallery badges
+            />
```

Comment:
Could also do this:

```
const galleryBadges = ["Black-Owned", "Women-Owned"]
```

But whatevs I don't feel strongly one way or the other very easy to change this stuff. Also I agree that it's not worth messing around in an admin interface. 👍